tests/integration/opup_test.py

```diff
@@ -27,8 +30,9 @@ def _dryrun(
         e.compile(pt.compiler.MAX_PROGRAM_VERSION),
         [],
         ExecutionMode.Application,
-        e.abi_argument_types(),
-        e.abi_return_type(),
+        e.method_signature(),
+        None,
+        None,
         txn_params=DryRunExecutor.transaction_params(
```

Comment:
👍  good move

Reply:
I need to carve out a separate PR here with possibly another PR in graviton. Just hacked this hoping the tests will pass.

Reply:
got it, ignore me. lmk if it goes to another PR :-)